clients/include/blas2/testing_gemv_batched.hpp

```diff
@@ -311,6 +316,7 @@ void testing_gemv_batched(const Arguments& arg)
                                                       batch_count);
         EXPECT_HIPBLAS_STATUS(
             actual, (invalid_size ? HIPBLAS_STATUS_INVALID_VALUE : HIPBLAS_STATUS_SUCCESS));
+	*/
         return;
     }
```

Comment:
Why not test with guard?   I would put in a TODO, but if you don't want double coverage put in a SKIPPED so these can be removed when seen in test run.